DP/primitiveCalculater.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
```

Comment:
Include a single libraray **#include <bits/stdc++.h>**
It will reduce number of lines & beneficial for space also.